### PR TITLE
Orams login logout

### PR DESCRIPTION
**Pull request by Blackfry**

### CODE REVIEW

**Comment by elisechant** [comment]
biggest PR ever! did you write Google Search engine in here?

File: apps/orams/actions/appActions.js [hunk 1]
```diff
@@ -36,3 +38,43 @@ export const setErrorMessage = errorMessage => ({
   type: SET_ERROR_MESSAGE,
   errorMessage
 })
+
+export function setAuthState(newState) {
+  return { type: SET_AUTH, newState }
+}
+
+export const fetchAuth = () => dispatch => {
+  dispatch(sendingRequest(true))
```
Comment (matt-sm) [inline apps/orams/actions/appActions.js]: we prob need to clear errors in all these types of actions

File: apps/shared/Login/LoginForm.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+            setFocus={setFocus}
+          />
+          <header className="page-heading page-heading-without-breadcrumb">
+            <h1>Sign in to the Marketplace</h1>
+          </header>
+          <p>
+            New to the Marketplace? <Link to={`${rootPath}/signup`}>Create your account.</Link>
+          </p>
+          <Form model={model} id="login" onSubmit={data => handleSubmit(data)}>
+            <p>
```
Comment (matt-sm) [inline apps/shared/Login/LoginForm.js]: should remove all these <p>

File: apps/orams/reducers/index.js [hunk 3]
```diff
@@ -1,8 +1,16 @@
 /* eslint-disable camelcase */
 import { combineReducers } from 'redux'
+import { createForms } from 'react-redux-form'
 
 import appReducer from './appReducer'
+import errorMessage from './errorMessage'
+import form_options from './form_options'
 
 export default combineReducers({
```
Comment (matt-sm) [inline apps/orams/reducers/index.js]: missing the fetchAuth dispatch

File: apps/orams/pages/HomePage.js [hunk 4]
```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import { withRouter, Switch, Route } from 'react-router-dom'
-import Homepage from 'orams/components/HomePage/HomePage'
+import Homepage from '../components/HomePage/HomePage'
```
Comment (marcoalfonso) [inline apps/orams/pages/HomePage.js]: Why change from absolute path?

File: apps/orams/pages/LoginPage.js [hunk 5]
```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import { withRouter, Redirect } from 'react-router-dom'
+
+import LoginForm from '../../shared/Login/LoginForm'
```
Comment (marcoalfonso) [inline apps/orams/pages/LoginPage.js]: Use absolute paths

File: apps/orams/pages/LogoutPage.js [hunk 6]
```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import { withRouter, Redirect } from 'react-router-dom'
+
+import BaseForm from '../../shared/form/BaseForm'
+import LoadingIndicator from '../../shared/LoadingIndicator/LoadingIndicator'
```
Comment (marcoalfonso) [inline apps/orams/pages/LogoutPage.js]: absolute paths

File: apps/orams/pages/HomePage.js (hunk 4 above)
Comment (Blackfry) [reply apps/orams/pages/HomePage.js]: It was failing on my branch, guess because my webpack wasn't setup. We can overwrite this.

**Pull request merged**